CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+## Bugs fixed
+
+* `middleware.debug`: use `resolve` with `&env` to fix debugging in case of a local shadows a var.
```

Comment:
```suggestion
* [#846](https://github.com/clojure-emacs/cider-nrepl/issues/846): `middleware.debug`: use `resolve` with `&env`, preventing exceptions if a local shadows a var.
```

Reply:
I've added link to the PR. Thanks.
But there were no exceptions, just a local shadowing.